modules/sanger-tol/nf-core-pipeline/repeats_bed.nf

```diff
@@ -0,0 +1,32 @@
+// Module that parses a repeat-masked genome (as Fasta) and extracts
+// the coordinates of the masked regions as a BED File
+process REPEATS_BED {
```

Comment:
Same as https://github.com/sanger-tol/insdcdownload/pull/2#pullrequestreview-1128369017

> The naming/location of `sanger-tol/nf-core-pipeline/repeats_bed.nf`. Why is this not under `local` or `sanger-tol/nf-core-modules`?

Reply:
Same as there. Will be addressed in a later version of the pipeline, once all the nf-core tooling for subworkflows is in place, with an upgrade to the 2.6 template, and even perhaps a merge of all the download pipelines ?